code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -360,6 +368,30 @@
 	if(volume >= 1)
 		T.wet_floor(2)
 
+/datum/reagent/cardox
```

Comment:
Get in touch with BygoneHero and possibly Skull132 regarding cardox - from what I can recall there was discussion and possibly work between those two regarding a more comprehensive and lore-friendly implementation.